skywalking/plugins/sw_mysqlclient.py

```diff
@@ -37,7 +37,9 @@ def install():
     def _sw_connect(*args, **kwargs):
         con = _connect(*args, **kwargs)
         con.host = kwargs['host']
-        con.db = kwargs['db']
+        if 'db' in kwargs:
```

Comment:
If there is no `db` in kwargs, what should we keep in the database name? I think if this is missing, the database would not show up in the database dashboard.

Reply:
Why are some in the `database` and the others in `bd`?

Reply:
I  think is the mysqlclient version. I  read the sourse code the lastast version is database. before is db.the connect method handel it like this to avoid version.

Reply:
I think is the mysqlclient version reason. In lastast, the param is database, but the other is db. so the source in mysqlclient to compatible.